### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by maoe**
This PR enables Travis CI with various GHC versions including the latest one.

* Currently the write-merge test fails, which is a known issue (#14)
    * We may or may not turn off this test until #14 is resolved.
* I dropped support for ancient GHC versions
    * 6.12.3 isn't supported by [hvr/multi-ghc-travis](https://github.com/hvr/multi-ghc-travis)
    * 7.4.1 [fails](https://travis-ci.org/maoe/ghc-events/jobs/200506545#L338-L344) to build the latest containers due to Safe Haskell errors. I didn't bother fixing this because I don't think we need to support that old GHC.

### CODE REVIEW

**Comment by Mikolaj** [comment]
Splendid! LGTM.

**Comment by maoe** [comment]
I now think it's better to disable write-merge for now because otherwise we have to check every travis build log to confirm nothing is actually broken.

**Comment by maoe** [comment]
Travis now skips write-merge: https://travis-ci.org/maoe/ghc-events/builds/200771369

**Comment by maoe** [comment]
Reworded the last commit message. s/is/are/

File: GhcEvents.hs [hunk 1]
```diff
@@ -13,8 +13,8 @@ import GHC.RTS.Events.Analysis.Capability
 import System.Environment (getArgs)
 import Data.Either (rights)
 import qualified Data.Map as M
-import System.IO (IOMode(ReadMode), openBinaryFile)
-import System.Exit (die)
+import System.IO
+import System.Exit
```
Comment (kvelicka) [inline GhcEvents.hs]: Why did you change these to non-selective imports? I see the case for `System.Exit` but not sure about `System.IO`.

**Comment by kvelicka** [comment]
Only posted an insignificant question on one of the changes. Looks good! 

@Mikolaj: shall I merge it or do we want someone else's opinion as well?

**Comment by Mikolaj** [comment]
IMHO, if @simonmar doesn't object, let's merge. Great job, BTW.

**Comment by kvelicka** [comment]
Right, pending on his call then.

File: GhcEvents.hs (hunk 1 above)
Comment (maoe) [inline GhcEvents.hs]: I removed the import list for `System.IO.stderr`, which is also used in `die`.

Comment (kvelicka) [inline GhcEvents.hs]: Oh, I missed that - fair enough.

**Comment by maoe** [comment]
Any objections?

**Comment by Mikolaj** [comment]
I think you can safely assume there are none. :)

**Comment by maoe** [comment]
Okay, could we merge then?

**Comment by Mikolaj** [comment]
O, you didn't have repo write access. My bad, I can actually add you. Done.

**Comment by maoe** [comment]
Great, thank you! I'm going to merge this then.

**Comment by maoe** [comment]
Could you please enable Travis builds? I see ghc-events on https://travis-ci.org/profile/haskell but it's grayed out so I cannot enable it.

**Comment by Mikolaj** [comment]
Right. Supposedly it should work now.

**Comment by maoe** [comment]
Thanks!

**Comment by Mikolaj** [comment]
You are welcome.
